cmd/call-endpoint/main.go

```diff
@@ -16,14 +17,15 @@ var args struct {
 	config.ConfigArgs
 	cmd.CommonOptions
 
-	Endpoint   string   `help:"Endpoint of the desired API." arg:""`
+	Endpoint   string   `help:"Endpoint of the desired API." arg:"" optional:""`
```

Comment:
Now that this is optional, there should be some sort of special case for handling when it is not specified.
Think about how you want to user to experience it.